pslab/external/MLX90614.py

```diff
@@ -1,46 +1,40 @@
-from __future__ import print_function
+from pslab.bus import I2CSlave
 
-
-def connect(route, **args):
-    return MLX90614(route, **args)
-
-
-class MLX90614():
+class MLX90614(I2CSlave):
+    _ADDRESS = 0x5A
+    _OBJADDR = 0x07
+    _AMBADDR = 0x06
     NUMPLOTS = 1
     PLOTNAMES = ['Temp']
-    ADDRESS = 0x5A
     name = 'PIR temperature'
 
-    def __init__(self, I2C, **args):
-        self.I2C = I2C
-        self.ADDRESS = args.get('address', self.ADDRESS)
-        self.OBJADDR = 0x07
-        self.AMBADDR = 0x06
+    def __init__(self):
+        super().__init__(self._ADDRESS)
 
-        self.source = self.OBJADDR
+        self.source = self._OBJADDR
 
         self.name = 'Passive IR temperature sensor'
         self.params = {'readReg': {'dataType': 'integer', 'min': 0, 'max': 0x20, 'prefix': 'Addr: '},
```

Comment:
I wonder what this is? It seems like it's related to the `readReg` method, specifying its argument as an integer that should be between 0 and 0x20. From the rest of the code I gather that there are two register addresses of interest: `_OBJADDR` (0x07) and `_AMBADDR` (0x06). Does the sensor have other register addresses that could be interesting?

Either way, this variable is unused and should be removed. The information about register address range (0-0x20) should be moved to a docstring if it is relevant to the user.